src/components/ui/Select/Select.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import { Select as UISelect } from '@faststore/ui'
+import type { SelectProps } from '@faststore/ui'
+import { CaretDown as CaretDownIcon } from 'phosphor-react'
+
+import './select.scss'
+
+type SelectOptions = {
+  [key: string]: string
+}
```

Comment:
Record<string, string> has the same meaning of this type, afaik.

Reply:
You're right. Done at https://github.com/vtex-sites/base.store/pull/299/commits/24470749d588f91ae1c0b6151c992c4ff3f2c5fa